deid/dicom/pixels/detect.py

```diff
@@ -142,9 +142,9 @@ def _has_burned_pixels_single(dicom_file, force, deid):
                 group_descriptions = [item.get("name", "")]
 
             else:
+                group_flags = []  # evaluation for a single line
```

Comment:
I think what probably happened here, because it's such an obvious mistake, is that I originally had a set of flags per item, but decided to change it to be on the level of the group, and didn't move the definition outside of the loop as I should. Aside from that, no idea why I did that, maybe just tiredness and then lack of testing it.

Reply:
It happens to the best of us... and now, we have some test methods for it! 😃

Reply:
You're the best to work with @wetzelj ! Your work is always so thorough, not just with the fix, but tests too. You're by far the best random collaboration I've found on here! Maybe some day if there is some project that warrants it we can write a software paper proper for it. Or since you've been so helpful, I'd be happy to do most of the writing. Cheers!

Reply:
You've been great to work with too @vsoch!  This has been a fun effort.  At some point, I'll share how we're utilizing deid. 
We're still in the initial development of our end-user application, but are close to having something to be able to start end-user testing! 

Have a great evening!